app/models/user.rb

```diff
@@ -48,30 +62,27 @@ def self.from_omniauth(auth)
       user.email = auth.info.email
       user.given_name = auth.info.given_name
       user.surname = auth.info.surname
+      user.display_name = auth.info.display_name
     end
   end
 
   # Definitely for sure wrong for some people. But staff want to be able to
   # sort on surname for processing purposes, so we're getting given name +
-  # surname.
+  # surname. This could pose a problem for those who prefer not to use 
+  # their legal surname. In an effort to make as few assumptions about 
+  # identity as possible, we are privileging preferred names for sorting.
   def name
-    if self.surname && self.given_name
+   if self.surname.present? && self.preferred_name.present?
```

Comment:
We're expecting preferred name to be in the format "Last, First M." (because it needs to match what is on the Thesis title page for authors) so for this method I don't think we need the first `if` clause. Let me know if that doesn't make sense or seems like it's going to cause problems.

Reply:
Thanks, Helen. That makes sense and helps a lot! Is the assumption that preferred name, if present, will replace given/first name?

Reply:
It will replace the entire name for display/metadata purposes. Basically, if preferred name is present it _is_ the display name. Otherwise the display name should be `Surname, Given M.`.